src/Popup/index.js

```diff
@@ -57,6 +59,7 @@ class Popup extends Component {
                 left: this.state.left,
                 top: this.state.top,
                 zIndex: this.state.zIndex,
+                display: this.isAnchorVisible ? '' : 'none',
```

Comment:
Looks like this will fail if popup is closed during `isAnchorVisible=false`, as `isAnchorVisible` will remains `isAnchorVisible` after next open.